lib/rules/selector-disallowed-list/README.md

```diff
@@ -115,3 +115,22 @@ The following pattern is _not_ considered a problem:
   .foo {}
 }
 ```
+
+### `ignore: ["keyframe-selectors"]`
+
+Ignore keyframe selectors.
+
+Given:
+
+```json
+["/from/", { "ignore": ["keyframe-selectors}"] }]
```

Comment:
Oops. 😅 

```suggestion
["/from/", { "ignore": ["keyframe-selectors"] }]
```

Reply:
Great catch, thanks!

(I should have caught it!)